models/chaincodes.js

```diff
@@ -0,0 +1,20 @@
+const firestore = require('../utils/firestore')
+const admin = require('firebase-admin')
+
+const chaincodeModel = firestore.collection('chaincodes')
+const storeChaincode = async (username) => {
+  try {
+    const userchaincode = await chaincodeModel.add({
+      username,

```

Comment:
@ojas-elawadhi , just curious, can't we simply use the ID of the users model as the chaincode? 🤔